monai/transforms/post/contour.py

```diff
@@ -0,0 +1,46 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import torch.nn as nn
+import numpy as np
+
+class Contour:
+    """
+    Return the contour of the organ in a mask image that only composes of 0 and 255 in gray value, 
+    with Laplace kernel as default.
+
+    Args:
+        kernel_type: the method applied to do edge detection.
+        to_resize: whether to keep original shape of `mask_image`
+    """
+
+    def __init__(self, kernel_type='Laplace', to_resize=False):
+        self.kernel_type = kernel_type
+        self.to_resize = to_resize
+
+    def __call__(self, mask_image):
+        '''
+        Args:
+        mask_image: torch tensor of the mask_image that you want to find the contour of,
+        with shape being (1, 1, width, height)
+        '''
+        if self.kernel_type != 'Laplace':
+            raise NotImplementedError
+
+        padding = 0 if self.to_resize else 1
+        conv = nn.Conv2d(1, 1, kernel_size=3, stride=1, padding=padding, bias=False)
+
+        kernel = np.array([[-1, -1, -1], [-1, 8, -1], [-1, -1, -1]], dtype='float32')
+        kernel = torch.from_numpy(kernel.reshape((1, 1, 3, 3)))
+        conv.weight = torch.nn.Parameter(kernel)
```

Comment:
should be: `conv.weight = torch.nn.Parameter(kernel, requires_grad=False)` to disable auto grad.

Reply:
sure, thanks